types/types.go

```diff
@@ -96,6 +98,9 @@ type FEVMQueries interface {
 	SimpleRamp() common.Address
 	// RateModule gets fetched from InfinityPool
 	RateModule() (common.Address, error)
+	// preview termination methods
+	PreviewTerminateSector(ctx context.Context, minerAddr address.Address, tipset string, vmHeight uint64, sectorNumber uint64, gasLimit uint64, offchain bool) (actor *filtypes.ActorV5, totalBurn *big.Int, epoch abi.ChainEpoch, err error)
+	PreviewTerminateSectors(ctx context.Context, minerAddr address.Address, tipset string, vmHeight uint64, batchSize uint64, gasLimit uint64, useSampling bool, optimize bool, offchain bool, maxPartitions uint64, errorCh chan error, progressCh chan *PreviewTerminateSectorsProgress, resultCh chan *PreviewTerminateSectorsReturn)
```

Comment:
I think we could probably keep these out of the SDK because they don't really involve the sdk at all. If we want to change an interface or a return value type then we'd have to be bothered with breaking changes

Reply:
Okay, I moved it out of the SDK interface, it now lives in: github.com/glifio/go-pools/terminate